cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,55 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import SetttingsObject from '../support/pages/settings.pageObject';
 
+const settingPageObject = new SetttingsObject();
+const { faker } = require('@faker-js/faker');
 describe('Settings page', () => {
+  const updateBio = faker.person.bio();
+  const updateName = faker.person.firstName();
+  const updateEmail = faker.internet.email();
+  const updatePssword = faker.internet.password();
```

Comment:
There's a typo in the variable name `updatePssword`. It should be `updatePassword`.